index.html

```diff
@@ -210,6 +210,71 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
         <abbr title="application programming interface">API</abbr> properties</a> in the [[core-aam-1.2]].
       </p>
     </section>
+    <section id="mapping_contextual">
+      <h3>Exposing HTML Features With Contextual Roles</h3>
+      <p>
+        An element with a <STRONG>contextual role</STRONG> is one whose implicit WAI-ARIA role semantics are based on its ancestral <a>accessibility tree</a> relationship to 
+        another element or elements. Or, an element whose computed role is reliant on whether the element has been provided an accessible name. 
```

Comment:
Do we want to start off with something even vaguer and more all encompassing, for example:

> An element with a contextual role is one whose implicit WAI-ARIA role semantics are dependent on the existence of other elements in the accessibility tree or the existence of certain attributes. 

Maybe that is too vague? Maybe is an "accessible name" an "attribute"? Can we imagine other attributes might effect a role? Like `tabindex`?

Regardless I do object to a sentence starting with `Or, `. Here is another try, just to try things out:

> An element with a contextual role is one whose implicit WAI-ARIA role semantics are effected by one or more of the following constraints:
> - A required ancestral relationship with other elements in the accessibility tree.
> - The existence of an accessible name.

Reply:
oh wow when I mentioned tabindex, I just remembered this had an effect on the "minimum role" concept. Honestly, it seems, minimum role is actually a "contextual role", and maybe... we don't need these things to be separately defined?

Reply:
i did have that thought as well... but the rational i kept coming back to as to why they should be separate is that contextual roles are elements that have defined roles, but do to both valid and invalid use, their role may need to adjust.  These behaviors largely exist now as either defined for specific elements, or are undefined behaviors that presently occur.

where as minimum roles are really only for elements that aren't meant to be anything more than generic or not exposed to the a11y api, but due to valid use of HTML, they need to have a minimum role based on the attribute that was used on the element.  minimum roles are not presently implemented, unlike most if not all of the contextual roles.

the distinctions could be clearer with further edits... but i do completely agree that the concepts are related and could be merged into a single section.  Just two different tracks, where i really don't want one concept to be held up by the other (minimum role being more important imo)

Reply:
ok I can easily see them as both separate or under the same umbrella, and I see your point about not holding things up.

Reply:
I like the first sentence phrasing of "ancestral" context, but I have a suggestion for the second to broaden it, but keep the name/label as an concise, understandable example.

```suggestion
        another element or elements. Or, an element whose computed role is dependent upon whether it has other associated accessibility information, such as an author-provided accessible label. 
```

Reply:
nice I like this suggestion, @cookiecrook